Sources/Fuzzilli/Core/CodeGenerators.swift

```diff
@@ -371,6 +371,31 @@ public let CodeGenerators: [CodeGenerator] = [
         b.reassign(target, to: val)
     },
 
+    CodeGenerator("DestructArrayGenerator", input: .iterable) { b, arr in
+        // TODO: We need a way to expose the length of arr so that element selection is accurate
```

Comment:
This'd be quite complex to do, and I don't think we really need it: 
```javascript
let arr = [1,2,3];
let [x, y] = arr;
// x = 1, y = 2
let [v1, v2, v3, v4] = arr;
// v4 = undefined
```
seems to work fine.

Maybe just update this comment to say something like "we could improve this logic by tracking the lengths of arrays and using that, but it's not strictly necessary since array destructuring allows the number of variables to be different from the length of the array"

Reply:
Actually, I can just drop the comment since it's strictly not necessary that we cap the array length.